etl/build.gradle

```diff
@@ -176,32 +199,48 @@ class ApiLoader {
     this.httpMethod = Method.POST
   }
 
-  void load() {
+  void load(boolean requireBrowseImg) {
     def successes = 0
     def failures = 0
+    def browseImgCollectionsFound = 0
+
     metadataUrls.eachWithIndex { metadata, i ->
+      println "Metadata request: ${i}"
       try {
         def metadataContent = new URL(metadata).text
-        target.request(httpMethod) {
-          requestContentType = 'application/xml'
-          body = metadataContent
-          response.success = { resp, data ->
-            successes++
-            println "${data.data.id} created #${i}: ${data.data.attributes.created}"
-            collections.add(data.data.id.replace('gov.noaa.nodc:', ''))
-          }
-          response.failure = { resp, data ->
-            failures++
-            println "Error uploading metadata #${i} from ${metadata}: ${data}"
-          }
+        def metadataMap = new XmlSlurper().parseText(metadataContent)
+        def idInfo = metadataMap.identificationInfo.MD_DataIdentification
+
+        // Check for browse graphic
+        def thumbnail = idInfo.graphicOverview.MD_BrowseGraphic.fileName.CharacterString.text()
+        println "Load metadata: ${(!requireBrowseImg || (requireBrowseImg && thumbnail?.trim()))}"
+        // browseImg not needed or it's required and its got an image
+        if (!requireBrowseImg || (requireBrowseImg && thumbnail?.trim())) {
+          browseImgCollectionsFound++
```

Comment:
We could probably rename this variable since one possibility is the collection doesn't require a thumbnail